### PR TITLE
dependency-check 9.0.0

### PR DESCRIPTION
**Pull request by jeremylong**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/d/dependency-check.rb [hunk 1]
```diff
@@ -11,7 +11,7 @@ class DependencyCheck < Formula
   end
 
   bottle do
-    sha256 cellar: :any_skip_relocation, all: "5c3d1cc94e4fe40f3f9624f0d9557b28519a42afda2e68393c98814a691f1d57"
+    sha256 cellar: :any_skip_relocation, all: "2fce32cfd7b0129bcaac37979cc6841e37b5c050e83c362fe64ff8fe716a6735"
```
Comment (chenrui333) [inline Formula/d/dependency-check.rb:14]: no need to change the bottle block

**Review by SMillerDev — APPROVED** [review]

File: Formula/d/dependency-check.rb (hunk 1 above)
Comment (jeremylong) [reply Formula/d/dependency-check.rb:14]:
```suggestion
    sha256 cellar: :any_skip_relocation, all: "5c3d1cc94e4fe40f3f9624f0d9557b28519a42afda2e68393c98814a691f1d57"
```

**Comment by jeremylong** [comment]
@SMillerDev I may have messed up the approval automation by pushing new changes...

**Review by chenrui333 — APPROVED** [review]

**Comment by Dhanxy** [comment]
Does it work for Java 11?

**Comment by jeremylong** [comment]
Yes, it works with Java 11.

**Pull request merged**